internal/pkg/source/source.go

```diff
@@ -109,13 +109,25 @@ func (a *Source) visitOperands(operands []*ssa.Value) {
 		if !ok || a.marked[n] {
 			continue
 		}
-		al, ok := (*o).(*ssa.Alloc)
-		if !ok || al.Comment == "slicelit" || a.config.IsSource(al.Type()) {
-			a.dfs(n)
+		al, isAlloc := (*o).(*ssa.Alloc)
+		// An Alloc represents the allocation of space for a variable. If a Node is an Alloc,
+		// and the thing being allocated is not a slice, then either:
+		// a) it is a Source value, in which case it will get its own DFS
```

Comment:
Could you please explain how it will get its DFS?

Reply:
Do you mean explain it in the comment or explain it here?

If the latter, it will get its own DFS when the code in `source.go:sourcesFromBlocks` catches it. The relevant line is line 239 in `source/source.go`.

Reply:
Thanks.
I think it would be useful to include this in the comment.

Reply:
Gotcha. Done.